framework/tlvf/test/main.cpp

```diff
@@ -152,6 +153,49 @@ int main(int argc, char *argv[])
     LOG(DEBUG) << "Done (WSC M2)";
     MAPF_INFO("TLV LENGTH WSC M2: " << thirdTlv->length());
 
+    auto fourthTlv     = msg.addClass<tlvTestVarList>();
```

Comment:
Consider putting each test in each own function - so there will be a TX method and RX method, will make `main` more readable and extensible